src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "http://purl.obolibrary.org/obo/BTO_0005588"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A non-ciliated secretory epithelial cell that is part of some oviduct.")
```

Comment:
Consider removing "secretory" from definition.
From PMID:22911892:
"These small, narrow cells have been described historically as non-functioning secretory cells that could possibly be the precursor of the secretory cell [[5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4442093/#R5)]; however, their role and function was not investigated."

Reply:
Some notes on text definition:
- The first sentence may describe other cell types (e.g., 'fallopian tube secretory epithelial cell').

- PMID:22911892 seems to only investigate the fallopian tube epithelium as opposed to whole oviduct. Is there a reference (other than BTO) that reports this cell in other locations of the oviduct?

- Since the name of this cell type (and the synonym) are based on morphological and positional features, consider noting this (small size, little cytoplasm, basally located, "found intercalated between the adjacent ciliated and secretory cells", etc.)

- In last sentence, capitalise letter of first word.